src/pytest_randomly/__init__.py

```diff
@@ -29,6 +29,14 @@
 except ImportError:  # pragma: no cover
     xdist = None
 
+# baker
```

Comment:
let's call it `model_bakery`, in comments, variable names, etc. and keep in order